website/content/api-docs/auth/approle.mdx

```diff
@@ -501,12 +512,21 @@ Assigns a "custom" SecretID against an existing AppRole. This is used in the
 - `token_bound_cidrs` `(array: [])` - Comma-separated string or list of CIDR
   blocks; if set, specifies blocks of IP addresses which can use the auth tokens
   generated by this SecretID. Overrides any role-set value but must be a subset.
+- `num_uses` `(integer: 0)` - Number of times this SecretID can be used, after which
+  the SecretID expires. A value of zero will allow unlimited uses.
+  Overrides secret_id_num_uses role option when supplied.
+- `ttl` `(string: "")` - Duration in seconds (`3600`) or an integer time unit (`60m`)
+  after which this SecretID expires. Defaults to 0, meaning no expiration.
+  Overrides secret_id_ttl role option when supplied.
 
 ### Sample Payload
 
 ```json
 {
-  "secret_id": "testsecretid"
+  "secret_id": "testsecretid",
+  "metadata": "{ \"tag1\": \"production\" }",
```

Comment:
can we keep the changes to what has actually changed please?

Reply:
I didn't think too much of this, but changed and pushed!